### PR TITLE
Add Jetbrains Space documentation with example

### PR DESCRIPTION
**Pull request by Sironheart**
In correspondance with the following discussion/ idea https://github.com/jreleaser/jreleaser/discussions/1426

### CODE REVIEW

File: docs/modules/continuous-integration/pages/jetbrains-space.adoc [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                "{{ run:number }}/helloworld-1.0.0-SNAPSHOT.jar"
+            )
+            localPath = "helloworld.jar"
+        }
+
+        // # icon:rocket[] Execute JReleaser
+        shellScript {
+            interpreter = "/bin/bash"
+            content = """
+                JRELEASER_SPACE_TOKEN = ${'$'}JB_SPACE_CLIENT_TOKEN
```
Comment (aalmiray) [inline docs/modules/continuous-integration/pages/jetbrains-space.adoc:67]: Where is this var required?

File: docs/modules/continuous-integration/pages/jetbrains-space.adoc [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+    // Use the predefined JReleaser Docker image
+    container(displayName = "jReleaser", image = "jreleaser/jreleaser-slim:latest") {
+        // use artifact from build step
+        fileInput {
+            source = FileSource.FileArtifact(
+                "{{ run:file-artifacts.default-repository }}",
+                "{{ run:number }}/helloworld-1.0.0-SNAPSHOT.jar"
+            )
+            localPath = "helloworld.jar"
```
Comment (aalmiray) [inline docs/modules/continuous-integration/pages/jetbrains-space.adoc:60]: I'd suggest adding a note here stating that this file must be in a relative path as expected by the JReleaser configuration

File: docs/modules/continuous-integration/pages/jetbrains-space.adoc [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    // limit release to git tags
+    startOn {
+        gitPush {
+            anyTag()
+        }
+    }
+
+    // allow use for environment variables
+    parameters {
+        secret("gpg-passphrase", value = "{{ project:gpg-passphrase }}")
```
Comment (aalmiray) [inline docs/modules/continuous-integration/pages/jetbrains-space.adoc:47]: Please add a note stating:
- this is how secretes may be passed to the execution
- this particular example shows how signature related to signing may be used

**Review by aalmiray — CHANGES_REQUESTED** [review]

File: docs/modules/continuous-integration/pages/jetbrains-space.adoc (hunk 1 above)
Comment (Sironheart) [reply docs/modules/continuous-integration/pages/jetbrains-space.adoc]: Nowhere yet. Probably was a little too motivated ;)

File: docs/modules/continuous-integration/pages/jetbrains-space.adoc (hunk 2 above)
Comment (Sironheart) [reply docs/modules/continuous-integration/pages/jetbrains-space.adoc:59]: added a comment for that.

File: docs/modules/continuous-integration/pages/jetbrains-space.adoc (hunk 3 above)
Comment (Sironheart) [reply docs/modules/continuous-integration/pages/jetbrains-space.adoc:45]: did so

**Review by aalmiray — APPROVED** [review]

**Pull request merged**